dodola/tests/test_services.py

```diff
@@ -32,6 +32,18 @@ def _datafactory(x, start_time="1950-01-01"):
     return out
 
 
+@pytest.fixture(scope="function")
+def domain_file(request):
+    """ Creates a fake domain file for testing"""
```

Comment:
Another very minor, optional thing: This makes me think that the fixture creates a literal file on the local system. Might revise this docstr to instead mention it returns a Dataset (or is it a DataArray?).

Reply:
fair point! Updating